wesnoth-l.po

```diff
@@ -3155,8 +3177,9 @@ msgid ""
 "wilderness homes. They can bullseye ferocious swamp rats and track anything "
 "that moves in their territory."
 msgstr ""
-"Lovčí strávili jejich životy v lesích a bažinách jejich domova, divočiny. "
-"Zasáhnou i blechu a dokáží vystopovat cokoliv, co prošlo jejich teritoriem."
+"Lovčí strávili své životy v lesích a bažinách svého domova, divočiny. "
+"Zasáhnou i krysu, divočící v močálu, a dokáží vystopovat cokoliv, co prošlo "
```

Comment:
Řekl bych, že tady má být přívlastek těsný.
```suggestion
"Zasáhnou i krysu divočící v močálu a dokáží vystopovat cokoliv, co prošlo "
```

Reply:
Asi ano. Ono je to taková konstrukce. Protože otrocky přeloženo by bylo, že zasáhnou i divokou močálovou krysu, což by teda neznělo jako velké umění. Tak jsem tam musel naznačit, že ta divokost znamená, že se rychle hýbe, čímž se umění lukostřelby patřičně vypíchne.